Mac/Tools/plistlib_generate_testdata.py

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env python3
 
 from Cocoa import NSMutableDictionary, NSMutableArray, NSString, NSDate, NSNumber
-from Cocoa import NSPropertyListSerialization, NSPropertyListOpenStepFormat
+from Cocoa import NSPropertyListSerialization
+# from Cocoa import NSPropertyListOpenStepFormat
```

Comment:
In https://bugs.python.org/issue14455#msg190950 Ronald said:

> * There is no support for the old OpenStep format. That format is badly
  documented, has been deprecated for a long time, and writing it is 
  not supported by Apple's plist libraries.  The OpenStep format also 
  seems to be much more limited than the two modern ones.

Perhaps we could just delete this and the following line:

```py
 #    ('openstep', NSPropertyListOpenStepFormat),
```

Reply:
When I work on such large "cleanup" patches, I really fear regressions. While your proposition makes sense, IMHO it's out of the scope of the PR. I don't know this code, I don't know how to test it. So I'm too shy to modify. If you mind, please write a separated PR for that.

Here I chose to copy the coding style of the file: keep the code but comment it...

Reply:
That line was commented-out since the first commit in https://github.com/vstinner/cpython/commit/c5cf7973422dce0ed59849aaf2d708d4c6b7320d#diff-39dd7d4b956a14a62f631dfb4c5f3fd4R14 but it's ultimately your call :) `plistlib_generate_testdata.py` was only modified by Ronald and @serhiy-storchaka: https://github.com/vstinner/cpython/commits/e0d2720ef5eccd9edb1ad82fc812b3a88e41a3fa/Mac/Tools/plistlib_generate_testdata.py And there are only 4 commits prior to your change. I'd say it's safe to remove it in this PR.

If you decide to keep it, then I'd suggest deleting `NSPropertyListOpenStepFormat` the import instead of commenting out it.

Reply:
Left this on to Ronald.

Reply:
Deleting the line NSPropertyListOpenStepFormat is fine.

Reply:
> Deleting the line NSPropertyListOpenStepFormat is fine.

Ok, done.